app/src/main/res/values/strings.xml

```diff
@@ -18,7 +18,7 @@
     <!-- Title for private session option -->
     <string name="private_browsing_title">You\'re in a private session</string>
     <!-- Explanation for private browsing displayed to users on home view when they first enable private mode -->
-    <string name="private_browsing_explanation">Fenix clears your search and browsing history when you close your private session. While this doesn\'t make you anonymous to websites or your internet service provider, it makes it easier to keep what you do online private from anyone else who uses this device. Common myths about private browsing</string>
+    <string name="private_browsing_explanation">Fenix clears your search and browsing history when you close your private session. While this doesn\'t make you anonymous to websites or your internet service provider, it makes it easier to keep what you do online private from anyone else who uses this device.\n\nCommon myths about private browsing</string>
```

Comment:
Does it make sense to split this into two strings and assemble them in code? I'm pretty certain you will end up with translations that do not contain `\n\n` like you expect. :)